### PR TITLE
dependency updates

### PR DESCRIPTION
**Pull request by jakobbraun**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,19 +4,19 @@
...
-    <version>2.1.1</version>
+    <version>2.1.2</version>
     <name>Virtual Schema for document data in files on AWS S3</name>
     <description>Adapter for document data access from files from AWS S3.</description>
-    <url>https://github.com/exasol/s3-document-files-virtual-schema</url>
+    <url>https://github.com/exasol/s3-document-files-virtual-schema/</url>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>11</java.version>
```
Comment (kaklakariada) [inline pom.xml:14]:
```suggestion
```
Already defined in parent pom

File: pom.xml [hunk 2]
```diff
@@ -381,152 +222,40 @@
...
-                        <configuration>
-                            <addOutputDirectory>false</addOutputDirectory>
-                            <sourceDirectory>src/test/java</sourceDirectory>
-                            <outputDirectory>target/delombok/test</outputDirectory>
-                        </configuration>
-                    </execution>
-                </executions>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <configuration>
+                    <forkCount>0</forkCount>
```
Comment (kaklakariada) [inline pom.xml:250]:
```suggestion
                    <useModulePath>false</useModulePath>
```
Please check if this works. I think this solves the root cause.

File: pom.xml [hunk 3]
```diff
@@ -4,19 +4,19 @@
...
-        <jacoco.version>0.8.7</jacoco.version>
+        <jacoco.version>0.8.8</jacoco.version>
         <gpg.skip>true</gpg.skip>
-        <org.testcontainers.version>1.16.3</org.testcontainers.version>
-        <vs-commom-document-files.version>6.0.1</vs-commom-document-files.version>
+        <org.testcontainers.version>1.17.1</org.testcontainers.version>
+        <vs-commom-document-files.version>7.0.0</vs-commom-document-files.version>
         <!--
                 M4 does not fail tests if beforeAll fails
                 M5 fails in integration tests with a strange error
```
Comment (kaklakariada) [inline pom.xml:22]: Can't comment on the next line, but I think you can also delete the `surefire.and.failsafe.plugin.version` property

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 4]
```diff
@@ -4,19 +4,19 @@
...
-    <url>https://github.com/exasol/s3-document-files-virtual-schema</url>
+    <url>https://github.com/exasol/s3-document-files-virtual-schema/</url>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>11</java.version>
         <junit.version>5.8.2</junit.version>
-        <jacoco.version>0.8.7</jacoco.version>
+        <jacoco.version>0.8.8</jacoco.version>
         <gpg.skip>true</gpg.skip>
```
Comment (kaklakariada) [inline pom.xml:17]:
```suggestion
```

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (jakobbraun) [reply pom.xml]: indeed. that seems to solve it.

Comment (jakobbraun) [reply pom.xml]: :+1:

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**